### ISSUE TITLE
Add soft cutoff

### ISSUE THREAD

**Issue by kubantjan** [mild-blue/txmatching#929]
Add soft cutoff parameter to configuration

Send another set of antibodies to FE that have MFI between cutoff  and soft cutoff with their crossmatch calculated.

**Comment by kristinagalik** [comment]
> * I am asking here comment from @kristinagalik  and @NotLubo  whether my proposed placement of antibodies makes sense to them (related FE issue [Show soft match antibodies #943](https://github.com/mild-blue/txmatching/issues/943) )

If it is supposed to be a new column or smth, then I think its fine. Maybe @NotLubo can still confirm this though. 

> Is that what you mean?

I would rather filter from `recipient.hla_antibodies.hla_antibodies_per_groups...` (you can see that it is accessed even in `get_crossmatched_antibodies`) because you can have `recipient.hla_antibodies.hla_antibodies_raw_list` empty. (why? no idea, dont have much time now, i can try to find out why this happens in the afternoon)

**Comment by NotLubo** [comment]
I think that another column in situations, like presented in screenshot, would substantially decrease the readability. Better approach would be the think of way to distinguish them in present column – maybe by dividing them in to 2 groups, changing the color or perhaps both.

<img width="797" alt="image" src="https://user-images.githubusercontent.com/40338867/180733391-950d1512-0e40-4516-b155-958c5c0a3a92.png">

**Comment by kubantjan** [comment]
@NotLubo yep, here i was not after the design, more after the data format sent ot frontend, whether the proposed one will be ok. Otherwies I aggree. Maybe best would be to  show just the ones that have some crossmatch in some less serious color (orange for example)

**Comment by NotLubo** [comment]
Well if divide antibodies in 2 seperate arrays than if we wanted to show them in diferent colors we would have to join and sort them again in FE. I think we should keep all antibodies in one array and set them property `soft_match` or maybe we could use `match_type`?

**Comment by kubantjan** [comment]
hmm extra property sounds like a good idea. its not really soft much but more like soft_antibody or smt like that. 

ok this sounds better @jogobeny  could you do it?

meaning adding this antibody into the antibodies we are sending now, just with flag soft_antibody=true?

**Comment by kubantjan** [comment]
EDIT. 
back to the trees! Sorry.
finally we have discussed this in detail today with @kristinagalik and think that in the end it will be better to send the soft crossmatch antibodies in a separate array. Because we will never want to sort them toghether with the normal antibodies. We will want to put them behind the normal antibodies (maybe differ them by color or put them behind some line, I do not know yet, but surely we will not want to mix the normal and soft (so the argument of @NotLubo  that the ordering on FE would be needed is not valid)

**Comment by kubantjan** [comment]
@jogobeny 

- yes, high res can in soft  only occur in the case when donor typization and recipient antibodies are both in high res and are the same

- we do not want NONE type in soft crossmatch.

**Comment by kubantjan** [comment]
presouvam do backlogu, ikem uzitecnost rozporoval, tak to neni uplne priorita..
